### DISCUSSION TITLE
GenyMotion virtual machine crashes (automatically shut down) when using Image Widget in Flutter

### DISCUSSION THREAD

**Discussion by chung19** [Genymobile/feedback#16] [Genymotion Desktop] [answered]
GenyMotion virtual machine crashes (automatically shut down) when using Image Widget in Flutter if the virtual machine uses a version higher than Android 5.1 please provide patch updates as soon as possible
![image](https://user-images.githubusercontent.com/98091082/183108536-d67cd703-8be1-4e45-98cf-844928f7fc5e.png)

**Comment by agavignet** [comment, marked as answer]
Hi @chung19

Sorry to hear about that. Could you provide us a minimal code sample using Flutter which demonstrates your issue (or a Github project), or an APK? This would help the technical team to investigate this issue.

Thank you,
Best regards
